### PR TITLE
allow placeholders in extensibility

### LINKED ISSUE #1782: EXT: Placeholders
**Description**
Allow user to make a placeholder in form ext configuration.

**Reasons**

**Attachments**

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:
- added placeholders to generic/simple list and string/number renderer
- ...
- ...

**Related issue(s)**
Closes #1782

### CODE REVIEW

**Review by dariadomagala — DISMISSED** [review]

File: docs/extensibility/form-widgets.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ Text widgets render a field as a text field. They are used by default for all st
 
 <img src="./assets/form-widgets/Text.png" alt="Example of a text widget" style="border: 1px solid #D2D5D9">
 
+#### Widget-specific parameters
+
+- **placeholder** - specifies a short hint that describes the expected value of an input field.
```
Comment (grego952) [inline docs/extensibility/form-widgets.md:39]:
```suggestion
- **placeholder** - specifies a short hint about the input field value.
```
The same applies to remaining placeholder definitions.

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**